bin/main.rb

```diff
@@ -0,0 +1,262 @@
+# rubocop:disable Metrics/CyclomaticComplexity
+# rubocop:disable Metrics/PerceivedComplexity
+# rubocop:disable Style/RedundantSelf
+# rubocop:disable Metrics/AbcSize
+# rubocop:disable Style/ParenthesesAroundCondition
+# rubocop:disable Style/GlobalVars
+# rubocop:disable Style/NumericPredicate
+# rubocop:disable Style/RedundantInterpolation
+# rubocop:disable Lint/RedundantCopDisableDirective
+# rubocop:disable Style/ConditionalAssignment
+# rubocop:disable Layout/LineLength
+# rubocop:disable Style/WhileUntilDo
+# rubocop:disable Lint/RedundantCopDisableDirective
+# rubocop:disable Style/GuardClause
+# rubocop:disable Style/IfUnlessModifier
```

Comment:
Find a way to fix Rubocop detected errors instead of disabling them, it helps you become a better developer. Two is okay but not all as you have added.